### ISSUE TITLE
Config files location

### ISSUE THREAD

**Issue by szotsaki** [savoirfairelinux/opendht#199]
1.3.4 version of opendht puts config files to /usr/etc instead of /etc (`CMAKE_INSTALL_FULL_SYSCONFDIR` instead of `CMAKE_INSTALL_SYSCONFDIR`).

Thought it's valid, the following warnings came up during package creation:

```
[   82s] opendht.x86_64: W: suse-filelist-forbidden-fhs23 /usr/etc is not allowed in FHS 2.3
[   82s] see http://www.pathname.com/fhs/ for a better location
[   82s]
[   82s] opendht.x86_64: W: standard-dir-owned-by-package /usr/etc
[   82s] This package owns a directory that is part of the standard hierarchy, which
[   82s] can lead to default directory permissions or ownerships being changed to
[   82s] something non-standard.
[   82s]
[   82s] opendht.x86_64: W: non-etc-or-var-file-marked-as-conffile /usr/etc/dhtnode.conf
[   82s] opendht.x86_64: W: non-etc-or-var-file-marked-as-conffile /usr/etc/dhtcluster.conf
[   82s] A file not in /etc or /var is marked as being a configuration file. Please put
[   82s] your conf files in /etc or /var.
```
`standard-dir-owned-by-package` was needed because this is the very first package which uses /usr/etc (it's empty on my computer, too).

If you must use this location, please ignore this bug report.

**Comment by aberaud** [comment]
Not sure if this is an OpenDHT bug or a Suse packaging/configuration issue.
On Debian CMAKE_INSTALL_FULL_SYSCONFDIR means /etc even when the prefix is /usr. Those variables are coming directly from the CMake' GNUInstallDirs plugin.

**Comment by aberaud** [comment]
Looks like CMake only have this behavior since v3.4. It indeed used to perpend the full prefix before this version. What version of CMake do you use ?

**Comment by szotsaki** [comment]
I create packages for [different versions](https://build.opensuse.org/repositories/home:szotsaki/opendht) of openSUSE, therefore different versions of CMake are used.

openSUSE Leap 42.1:  3.3.2
openSUSE Leap 42.2:  3.5.2
openSUSE Leap 42.3 [next major release]:  3.5.2
openSUSE Tumbleweed:  3.8.2

**Comment by aberaud** [comment]
Seems to be a CMake/GNUInstallDirs issue that was fixed in 3.4, so it should only affect openSUSE Leap 42.1 (with CMake 3.3.2).

A packaging fix might be to use the flag `-DCMAKE_INSTALL_SYSCONFDIR=/etc`. Would that be a reasonable solution ?

**Comment by szotsaki** [comment]
Thank you, with defining that variable it works as intended.
